RequestService/RequestService.Handlers/PutUpdateShiftStatusToAcceptedHandler.cs

```diff
@@ -0,0 +1,112 @@
+﻿using MediatR;
+using System.Threading;
+using System.Threading.Tasks;
+using RequestService.Core.Interfaces.Repositories;
+using RequestService.Core.Services;
+using HelpMyStreet.Contracts.RequestService.Request;
+using HelpMyStreet.Contracts.RequestService.Response;
+using HelpMyStreet.Contracts.CommunicationService.Request;
+using HelpMyStreet.Utils.Enums;
+using HelpMyStreet.Utils.Models;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace RequestService.Handlers
+{
+    public class PutUpdateShiftStatusToAcceptedHandler : IRequestHandler<PutUpdateShiftStatusToAcceptedRequest, PutUpdateShiftStatusToAcceptedResponse>
+    {
+        private readonly IRepository _repository;
+        private readonly ICommunicationService _communicationService;
+        private readonly IJobService _jobService;
+        public PutUpdateShiftStatusToAcceptedHandler(IRepository repository, ICommunicationService communicationService, IJobService jobService)
+        {
+            _repository = repository;
+            _communicationService = communicationService;
+            _jobService = jobService;
+        }
+
+        public async Task<PutUpdateShiftStatusToAcceptedResponse> Handle(PutUpdateShiftStatusToAcceptedRequest request, CancellationToken cancellationToken)
+        {
+            PutUpdateShiftStatusToAcceptedResponse response = new PutUpdateShiftStatusToAcceptedResponse()
+            {
+                Outcome = UpdateJobStatusOutcome.Unauthorized,
+                JobID = -1
+            };
+
+            int existingJobId = await _repository.VolunteerAlreadyAcceptedShift(request.RequestID, request.SupportActivity.SupportActivities.First(), request.VolunteerUserID, cancellationToken);
```

Comment:
Why SupportActivities.First()?  Feels like this method should just accept a single SupportActivity, rather than a collection.